civictechprojects/static/css/partials/_CreateProjectForm.scss

```diff
@@ -1,4 +1,12 @@
-//overrides a very particular user agent stylesheet declaration
+.form-body {
```

Comment:
Not blocking, but it might be worth making the classname less generic than form-body. create-form-body? Or sidestep it entirely; targeting `.create-form h1` and `.create-form h2` would probably do fine as well - whatever makes the most sense to you.

Reply:
In this case I think generic is best because the design team's mockups for other planned creation pages have the same header layout.